File: Model/ViewModel/OrderOpinionVM.cs
using System.Collections.Generic;
using targeteo.pl.Model.ViewModel;

namespace targeteo.pl.Model.ViewModel
{
    public class OrderOpinionVM
    {
        public int Id { get; set; }

        public string Text { get; set; }
        public int OrderDetailId { get; set; }
        public int PictureId { get; set; }
        public int Evaluation { get; set; }

        public virtual OrderDetailVM OrderDetails { get; set; }

        public virtual PictureVM Pictures { get; set; }
    }
}

File: ClientApp/src/app/model/orderOpinionVm.ts
import { orderDetailVm } from "./orderDetailVm";
import { PictureVm } from "./pictureVm";

export class OrderOpinionVm {
    
    public Id: number;
    public Text: string;
    public OrderDetailId: number;
    public PictureId: number;
    public Evaluation: number;
    public OrderDetails: orderDetailVm;
    public Picture: PictureVm;
}